src/box/lua/call.c

```diff
@@ -490,13 +490,40 @@ box_process_lua(struct call_request *request, struct port *base,
 	port_lua_create(base, L);
 	((struct port_lua *) base)->ref = coro_ref;
 
+	/*
```

Comment:
Regarding ec9b544a123f3e0614ab94b83a7ac4b395326f34 backport:

The commit message and the pull request description should cleanly state a motivation **of the backport**, when it is different from a motivation of the original commit. Otherwise a reader has no chance to understand why the change was backported. It is very sad, when you have no change to gain knowledge what may go wrong if related code will be changed. I would even say that it may lead to a kind of cargo cult, when some code is kept, because nobody knows, whether it could be changed.

We backport it due to two reasons:

* Make 1.10 and 2x code more similar to make further backports easier and more safe.
* Fill the fiber's Lua state in a frequent case to, again, don't accidentally run into a problem that doesn't exist on 2x.

The original commit was made as an optimization for merger. 1.10 has no merger module at all.

Feel free to choose any form of the description: you can remove the original commit message or keep it separately from backport's motivation. Just do it in a way that allows another developer to understand what is going on here. And don't forget about PR description: it seems, it doesn't reflect recent changes in the patchset.

While we're here:

> That may lead to a PANIC in `cord_on_yield`.

But the panic is a correct behavior. The difference is that before the patchset we had an assertion fail or a memory access problem in the case, but now tarantool prints a backtrace and panics.